packages/lesswrong/lib/routes.ts

```diff
@@ -373,155 +373,169 @@ addRoute(
   }
 );
 
-if (taggingNameIsSet.get()) {
+
+const usePluralTagName = !isLWorAF && taggingNameIsSet.get();
+const taggingNameCapitalized = usePluralTagName ? taggingNamePluralCapitalSetting.get() : taggingNameCapitalSetting.get();
```

Comment:
This should probably be structured as fake setting with a getter, like the others (in `instanceSettings.ts`); I don't think instanceSettings can change on the fly like db settings but not worth the risk.